org.eclipse.lyo.oslc4j.build/pom.xml

```diff
@@ -41,6 +41,7 @@
     <module>../org.eclipse.lyo.oslc4j.wink</module>
     <module>../org.eclipse.lyo.oslc4j.registry</module>
     <module>../org.eclipse.lyo.repackaging.jersey</module>
+    <module>../org.eclipse.lyo.shacl</module>
   </modules>
```

Comment:
@jadelkhoury This line is new. Otherwise it is just a pure code move from Lyo Validation repository.